web/template/edit_complaint.gohtml

```diff
@@ -67,6 +67,36 @@
 
         {{ template "input-date" (field "resolutionDate" "Resolution date" .Complaint.ResolutionDate .Error.Field.resolutionDate) }}
 
+        <div class="govuk-form-group {{ if .Error.Field.compensationType }}govuk-form-group--error{{ end }}">
+          <label class="govuk-label" for="f-category">
+            Compensation
+          </label>
+          {{ template "errors" .Error.Field.compensationType }}
+
+          <div class="govuk-radios" data-module="govuk-radios">
+            {{ range $k, $v := .CompensationTypes }}
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="f-compensation-type-{{ $k }}" name="compensationType" type="radio" value="{{ $v.Handle }}" data-aria-controls="conditional-compensation-{{ $k }}" {{ if or (eq $v.Handle $.Complaint.CompensationType) (and (not $.Complaint.CompensationType) (eq $v.Handle "NOT_APPLICABLE")) }}checked{{ end }}>
+                <label class="govuk-label govuk-radios__label" for="f-compensation-type-{{ $k }}">
+                  {{ $v.Label }}
+                </label>
+              </div>
+              {{ if not (eq $v.Label "N/A") }}
```

Comment:
Should probably base this on the handle so that we can easily change the label:
```suggestion
              {{ if not (eq $v.Handle "NOT_APPLICABLE") }}
```